src/main/kotlin/org/jitsi/nlj/codec/vp8/Vp8Utils.kt

```diff
@@ -75,6 +79,64 @@ class Vp8Utils {
 
         fun getTemporalLayerIdOfFrame(vp8Packet: RtpPacket) =
             DePacketizer.VP8PayloadDescriptor.getTemporalLayerIndex(
-                vp8Packet.buffer, vp8Packet.payloadOffset, vp8Packet.length)
+                vp8Packet.buffer, vp8Packet.payloadOffset, vp8Packet.payloadLength)
+
+        /**
+         * Returns the delta between two VP8 extended picture IDs, taking into account
+         * rollover.  This will return the 'shortest' delta between the two
+         * picture IDs in the form of the number you'd add to b to get a. e.g.:
+         * getPictureIdDelta(1, 10) -> -9 (10 + -9 = 1)
+         * getPictureIdDelta(1, 32760) -> 9 (32760 + 9 = 1)
+         * @return the delta between two extended picture IDs (modulo 2^15).
+         */
+        @JvmStatic
+        fun getPictureIdDelta(a: Int, b: Int): Int {
```

Comment:
shall we name this getExtendedPictureIdDelta, to distinguish between handling 15 and 7 bits picture ids?

Reply:
Done.